app/main.py

```
@@ -231,6 +231,20 @@ async def health():
 
 def start_server(host: str = "0.0.0.0", port: int = 7014, local_dev: bool = False):
     """Start the Uvicorn server for the FastAPI application."""
+    # Parse command line arguments if called via poetry script
+    # This allows `poetry run api --local-dev` to work correctly
+    import sys
```

Ruff (0.14.10)

Redefinition of unused `sys` from line 6

Remove definition: `sys`

(F811)